src/synopsys-action/synopsys-bridge.ts

```diff
@@ -91,6 +93,23 @@ export class SynopsysBridge {
         cwd: workingDirectory
       }
       try {
+        if (inputs.ENABLE_NETWORK_AIR_GAP) {
+          info('bridge default path :'.concat(this.getBridgeDefaultPath()))
+          info('synopsys_bridge_path:'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          //const bridgepath = inputs.SYNOPSYS_BRIDGE_PATH.length == 0 ? this.getBridgeDefaultPath() : inputs.SYNOPSYS_BRIDGE_PATH
+          info('inputs.SYNOPSYS_BRIDGE_PATH.length::'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
```

Comment:
why are we validating like inputs.SYNOPSYS_BRIDGE_PATH.length !== 0, can't we check like if(inputs.SYNOPSYS_BRIDGE_PATH.length) 
is there any specific reason to check empty string with length